src/cli.ts

```diff
@@ -1,23 +1,58 @@
 import { Command, flags } from "@oclif/command";
+import { OutputFlags } from "@oclif/parser";
 import { readFile, outputFile, existsSync } from "fs-extra";
 import { join, relative, parse } from "path";
 import slash from "slash";
 import ts from "typescript";
 import { generate, GenerateProps } from "./core/generate";
-import inquirer from "inquirer";
 import {
-  configSchema,
+  tsToZodconfigSchema,
   getSchemaNameSchema,
   nameFilterSchema,
   TsToZodConfig,
+  createConfig,
+  Config,
 } from "./config";
 import { getImportPath } from "./utils/getImportPath";
 import ora from "ora";
+import prettier from "prettier";
 import * as worker from "./worker";
+import inquirer from "inquirer";
+import { eachSeries } from "async";
+
+// Try to load `ts-to-zod.config.js`
+// We are doing this here to be able to infer the `flags` & `usage` in the cli help
+const tsToZodConfigJs = "ts-to-zod.config.js";
+const configPath = join(process.cwd(), tsToZodConfigJs);
+let parsedConfig: ReturnType<typeof tsToZodconfigSchema.safeParse> | undefined;
+let haveMultiConfig = false;
+const configKeys: string[] = [];
+
+try {
+  if (existsSync(configPath)) {
+    const rawConfig = require(slash(relative(__dirname, configPath)));
+    parsedConfig = tsToZodconfigSchema.safeParse(rawConfig);
+    if (parsedConfig.success && Array.isArray(parsedConfig.data)) {
+      haveMultiConfig = true;
+      configKeys.push(...parsedConfig.data.map((c) => c.name));
+    }
+  }
+} catch {
+  console.log(`${tsToZodConfigJs} can't be loaded!`);
+}
```

Comment:
Expose the `error` here, this is in case `ts-to-zod.config.js` is there but not a valid js file